migrations/20191105164531_add_incrementing_id.js

```diff
@@ -0,0 +1,14 @@
+
+exports.up = function(knex) {
+  return knex.raw(`
+    ALTER TABLE establishments
+      ALTER COLUMN id ADD GENERATED BY DEFAULT AS IDENTITY (START WITH 2000001);
+  `);
```

Comment:
Using raw here because knex's `increment()` uses serials and not the newer (and more standard-y) IDENTITY type which was introduced in Postgres 10.